src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -307,25 +307,29 @@ public function updateStatusAction(int $orderId, Request $request): RedirectResp
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            try {
-                $this->getCommandBus()->handle(
-                    new UpdateOrderStatusCommand(
-                        $orderId,
-                        (int) $form->getData()['new_order_status_id']
-                    )
-                );
-
-                $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
-            } catch (Exception $e) {
-                $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
-            }
+            $this->handleOrderStatusUpdate($orderId, (int) $form->getData()['new_order_status_id']);
         }
 
         return $this->redirectToRoute('admin_orders_view', [
             'orderId' => $orderId,
         ]);
     }
 
+    /**
+     * Updates order status directly from list page.
+     *
+     * @param int $orderId
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function updateStatusFromListAction(int $orderId, Request $request): RedirectResponse
```

Comment:
Dont forget Security 😉

Reply:
yeah forgot about it - added it with this commit https://github.com/PrestaShop/PrestaShop/pull/15908/commits/a88d20b00ddd649c12a5e8d749fee992e4679712